index.js

```diff
@@ -127,6 +127,19 @@ function Fetch(url, opts) {
 			});
 		}
 
+		if (options.abortSignal) {
```

Comment:
This should be named just `signal`? https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/fetch

Reply:
That is correct, my bad (even though I don't approve of having the short name)